docs/guides/main.rst

```diff
@@ -93,24 +93,23 @@ Installing on Windows
 Installing beets on Windows can be tricky. Following these steps might help you
 get it right:
 
-1. If you don't have it, `install Python`_ (you want Python 3.6). On the last
-   screen, the installer gives you the option to "add Python to PATH." Check
-   this box. If you do that, you can skip the next step.
+1. If you don't have it, `install Python`_ (you want Python 3.6). During the 
+   installation, look for an option to "Add Python 3.6 to PATH". Check this box.
+   If you have done this, you can skip the next step.
```

Comment:
Thanks! I didn't realize this could be on different screens in the Windows installer.